components/logging/org.wso2.carbon.logging.service/pom.xml

```diff
@@ -75,6 +75,10 @@
             <artifactId>guava</artifactId>
         </dependency>
         <dependency>
+            <groupId>org.springframework.ws.wso2</groupId>
+            <artifactId>spring.framework</artifactId>
+        </dependency>
+        <dependency>
```

Comment:
This is to get "org.springframework.util.Log4jConfigurer" class to initialize logging.